Source/Public/Get-ADOPSOrganizationRepositorySettings.ps1

```diff
@@ -0,0 +1,17 @@
+function Get-ADOPSOrganizationRepositorySettings {
+    [CmdletBinding()]
+    param (
+        [Parameter()]
+        [string]$Organization
+    )
+
+    # If user didn't specify org, get it from saved context
+    if ([string]::IsNullOrEmpty($Organization)) {
+        $Organization = GetADOPSDefaultOrganization
+    }
+
+    $Uri = "https://dev.azure.com/$Organization/_api/_versioncontrol/AllGitRepositoriesOptions?__v=5"
```

Comment:
No need to change it, but the query string here doesn't appear to do anything. (Working with unsupported APIs is fun. Officially there is no `_api` endpoint - it's supposed to be `_apis`, but this api only works _without_ the `s` 😂